### ISSUE TITLE
IOTA With commands not found

### ISSUE THREAD

**Issue by abrahamSecmotic** [telefonicaid/iotagent-node-lib#599]
Hi! 

I was creating an IOTAgent with this library and I cannot find the part related to IOTA With commands. 

When I clicked in the link of the index there is no result for this match. Where can I solve this problem or find this documentation about this? 

Thanks!

**Comment by fgalan** [comment]
Thanks for the feedback!

Could you tell us the particular link in which particular file is "broken", please?

**Comment by abrahamSecmotic** [comment]
Yes, https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/howto.md#commands

Thanks!

**Comment by fgalan** [comment]
I'm afriad that links is broken... However, it seems documentation command is included in other parts of the documentation, for instance: https://github.com/telefonicaid/iotagent-node-lib#commands

I'd recommend you to have a look to README.md and the other documents in the doc/ directory searching for commands information.
